docs/extensions.md

```diff
@@ -111,7 +111,7 @@ python -m pip install myliteextension
 In most cases, this would mean adding the extension to a `requirements.txt` file:
 
 ```
-jupyterlite==0.1.0a12
+jupyterlite==0.1.0b4
```

Comment:
do we have this under lint right now?

Reply:
Do you mean bumping the version automatically in this file?

If so then it's not the case (but could be handled).

Otherwise we can just drop the version here:

```suggestion
jupyterlite
```

Reply:
Well, i'm more about version _linting_ than bumping. More so that most stuff, I think a version pin is going to be clutch.

So i'll update it to be "correct" (without a version) but add a note about _YOU SHOULD ALWAYS PIN THE VERSION_